<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="9,1">
   <item>
    <layout class="QHBoxLayout" name="hbox" stretch="8,2">
     <item>
      <widget class="QVideoWidget" name="vw" native="true"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Play List</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
        <item>
         <widget class="QListWidget" name="lw"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pb_add">
            <property name="text">
             <string>Add Media</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_del">
            <property name="text">
             <string>Del Media</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Play Menu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="hsld">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pb_play">
          <property name="text">
           <string>▶️</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_stop">
          <property name="text">
           <string>⏹</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_pau">
          <property name="text">
           <string>⏸</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_ff">
          <property name="text">
           <string>⏩</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_prev">
          <property name="text">
           <string> ⏪</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="dial"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">PyQt6.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
